src/proposals/mips/mip-o04/MTokens.json

```diff
@@ -0,0 +1,21 @@
+[
+    {
+        "initialMintAmount": 1e12,
+        "collateralFactor": 0.65e18,
+        "reserveFactor": 0.25e18,
+        "seizeShare": 0.03e18,
+        "supplyCap": 0,
+        "borrowCap": 0,
```

Comment:
This should be `4_500_000` - see https://github.com/moonwell-fi/moonwell-contracts-v2/pull/298/files#diff-901d2a741de73b7c66fe240e11a28382320fde30ae970842cd7208e08fcf16d6R28